src/studio/src/designer/backend/package.json

```diff
@@ -31,7 +31,7 @@
     "jquery-validation-unobtrusive": "3.2.12",
     "js-beautify": "1.14.0",
     "json-editor": "0.7.28",
-    "monaco-editor": "0.25.2",
+    "monaco-editor": "0.26.1",
```

Comment:
not sure if we should tamper with monaco-editor as it tends to break something when we upgrade...

Reply:
Verified working locally 👍